src/css/index.scss

```diff
@@ -1,2 +1,167 @@
 @import './global';
 @import './information';
+
+$basic-blue: #bcc8dc;
+$basic-grey: #edeff5;
+$dark-red: #d93347;
+$dark-blue: #314a74;
+
+.HOME {
+  position: fixed;
+  left: 70px;
+  top: 68px;
+  color: white;
+  font-size: 20px;
+  font-weight: bold;
+}
+.HOME:hover {
+  color: $basic-blue;
+}
+
+header {
+  height: 200px;
+}
+
+section ul {
```

Comment:
Which section is targeted for? The skills section? So you may need to use class name selector to make it more explicit.